diesel_cli/src/database.rs

```diff
@@ -429,6 +429,11 @@ fn change_database_of_url(
         .last()
         .expect("The database url has at least one path segment")
         .to_owned();
+
+    if database.eq("postgres") {
```

Comment:
I've read the issue and I understand it may be sensible to do this specifically when we are trying to drop a database.
It also sounds likely that at the moment this is the only place at which this function is called.
However the naming of this function  `fn change_database_of_url(` would not let any future code author expect this behavior, so it's likely to introduce a bug.
I think it would be best to put this logic where the function is called, that is in a place where the current code path is about removing the database.

Reply:
Thanks for the review! I get what you mean, will push a commit with the required change.